src/elements/content-sidebar/activity-feed/comment/BaseComment.js

```diff
@@ -168,22 +128,13 @@ export const BaseComment = ({
         commentFormSubmitHandler();
     };
 
-    const handleStatusUpdate = (selectedStatus: FeedItemStatus): void => {
-        if (onAnnotationEdit) {
-            onAnnotationEdit({ id, permissions });
-        } else if (onCommentEdit) {
-            onCommentEdit({ id, status: selectedStatus, hasMention: false, permissions });
-        }
-    };
-
     // Since we have to pass onCommentEdit through annotations (to Replies), onAnnotationEdit essentially overrides onCommentEdit
     const onEdit = onAnnotationEdit ?? onCommentEdit;
 
-    const canDelete = permissions.can_delete;
-    const canEdit = onEdit !== noop && permissions.can_edit;
-    const canResolve = onEdit !== noop && permissions.can_resolve;
-    const createdAtTimestamp = new Date(created_at).getTime();
-    const createdByUser = created_by || PLACEHOLDER_USER;
+    const canDelete = permissions.can_delete ?? false;
```

Comment:
We should be cautious about changing business logic in a refactor. Are these values nullable now? If not, is the `?? false` needed?

Reply:
In this case, we really just need a boolean. It is not necessary to continue to allow it to be nullable after this point.

Reply:
Maybe actually the better syntax would be `!!can_delete`, come to think of it.